acceptance-test/docker.test.ts

```diff
@@ -0,0 +1,19 @@
+import { execSync } from 'child_process'
+import { argv } from 'node:process'
+import { exit } from 'process'
+
+const dockerTag = argv.filter((x) => x.startsWith('--containerTag='))[0]?.split('=')[1]
```

Comment:
dockerTag wrapped in quotes and while exec the docker run comand there is an error of invalid reference format for docker.
maybe this problem is only on windows machines

```suggestion
const dockerTag = argv.filter((x) => x.startsWith('--containerTag='))[0]?.split('=')[1].replace(/['"]+/g, '')
```

Reply:
@iryna-markovich-vocovo yep, on MAC is fine. I will correct it.